src/expressions/loop-expr.md

```diff
@@ -61,6 +89,11 @@ while let Some(y) = x.pop() {
 
 ## Iterator loops
 
+> **<sup>Syntax</sup>**  
+> _IteratorLoopExpression_ :  
+> &nbsp;&nbsp; `for` _Pattern_ `in` [_Expression_]<sub>except struct expression</sub>
```

Comment:
Like `if`, we probably want to link to struct expressions here.

Reply:
Still deciding how I'll do this (the "no struct expression" thing) to make it clear. But yeah, I agree a link to it will be needed.